app/models/user.py

```diff
@@ -40,6 +41,18 @@ def isBonnerScholar(self):
     @property
     def fullName(self):
         return f"{self.firstName} {self.lastName}"
+    
+    @property
+    def getEmergencyContact(self):
```

Comment:
Since this is a property the function should not be named `get...`, it should be descriptive as to what it is. Like above, its not `getFullName` but `fullName`.